r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -139,17 +140,6 @@ private CarePlan resolveActions(Session session) {
         return session.getCarePlan();
     }
 
-    public List<Resource> getAllContainedResources(Resource resource) {
-        if (!(resource instanceof DomainResource)) {
-            return new ArrayList<>();
-        }
-        List<Resource> contained = ((DomainResource) resource).getContained();
-
-        return Stream
-            .concat(contained.stream(), contained.stream().flatMap(r -> getAllContainedResources(r).stream()))
```

Comment:
Now that the references to Collectors and Stream were moved out of this file can you remove the unused imports?